PixiEditor/Models/UserPreferences/PreferencesSettings.cs

```diff
@@ -107,5 +118,54 @@ public static void AddCallback(string setting, Action<object> action)
                 ? (T)Preferences[name]
                 : fallbackValue;
         }
+
+        public static T? GetLocalPreference<T>(string name)
+        {
+            return GetPreference(name, default(T));
+        }
+
+        public static T? GetLocalPreference<T>(string name, T? fallbackValue)
+        {
+            if (IsLoaded == false)
+            {
+                Init();
+            }
+
+            return LocalPreferences.ContainsKey(name)
+                ? (T)LocalPreferences[name]
+                : fallbackValue;
+        }
+
+#nullable disable
+
+        private static string GetPathToSettings(Environment.SpecialFolder folder)
+        {
+            return Path.Join(
+            Environment.GetFolderPath(folder),
+            "PixiEditor",
+            "user_preferences.json");
```

Comment:
This is a minor thing, but for settings that are generated by the editor, I would change the name to `editor_data.json` or something like that